pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -514,6 +514,13 @@ type CommonServiceClassSpec struct {
 	// ServiceInstance provisioned from this ServiceClass will not
 	// work correctly.
 	Requires []string `json:"requires,omitempty"`
+
+	// DefaultProvisionParameters are default parameters passed to the broker
+	// when an instance of this class is provisioned. Any parameters defined on
+	// the plan and instance are merged with these defaults, with
+	// plan and then instance-defined parameters taking precedence the class
```

Comment:
"taking precedence the class defaults" - - missing the word "over" before "the class"